src/pyocr/tesseract.py

```diff
@@ -168,7 +168,7 @@ def detect_orientation(image, lang=None):
         original_output = original_output.strip()
         try:
             output = original_output.split("\n")
-            output = [line.split(": ") for line in output if (": " in line)]
+            output = [line.split(": ",1) for line in output if (": " in line)]
```

Comment:
Thanks for the hint! Never wrote python before, so I wasn't really aware of any conventions.